action.yml

```diff
@@ -13,6 +13,9 @@ inputs:
   source:
     description: Path to the directory containing the Go source to be published. This folder should contain at least a go.mod file.
     required: true
+  files:
+    description: List of glob patterns to select Go source to be published. Defaults to "**" (all files).
+    default: "**"
```

Comment:
If we're only caring about go sources here, why don't we use `["*.go", "go.mod", "go.sum"]` as a default?
Wondering if we could end up publishing unwanted changes with this default

Reply:
I'm planning to make this the default value we set via ci-mgmt, but I think it's least suprising for this action to just publish everything in the selected folder by default if you've not asked to filter anything out.

Reply:
I guess you're right! It's probably more surprising if we run into "why are those files not published" issues